docs/release.rst

```diff
@@ -14,6 +14,9 @@ Unreleased
 Maintenance
 ~~~~~~~~~~~
 
+* Remove Python 2 code
+  By :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>` :issue:`368`.
+
```

Comment:
Would it be possible to move this after the `cpuinfo` entry? That should save us a merge conflict (from the Python 3.11 PR)

Reply:
Fixed